### ISSUE TITLE
Jiralert fails to re-open closed tickets.

### ISSUE THREAD

**Issue by RobotLimeLtd** [free/jiralert#50]
I can’t get Jiralert to re-open tickets in status “Done”.  Instead it simply creates a new ticket, with a warning:

`notify.go:74] No issue matching ALERT{alert name=“TestAlert”} found, creating new issue`

However, if I close an issue with resolution “Won’t Fix”, the logs show that it found the ticket and knew not to create a new issue.  Also, when the issue is first created, Jiralert does not create new tickets while it’s in status Open or In Progress.

In summary - everything works except the re-opening of closed tickets.

Any idea what’s going wrong?

**Comment by RobotLimeLtd** [comment]
Ok - ignore; I’d been confused by this entry in the example config...

```
#Optional (default: always reopen)
reopen_duration: 0h 
```

If you leave this setting in place, or remove that option, the default is actually to **always create a new ticket**.

If you want tickets to be re-opened, then you have to configure a positive value, such as “24h”.

I haven’t tried this, but for “always reopen” functionality, you could set a ridiculously high value of hours.

**Comment by free** [comment]
> I haven’t tried this, but for “always reopen” functionality, you could set a ridiculously high value of hours.

Or, indeed, years.
